docker-compose.yml

```
@@ -38,6 +38,21 @@ services:
-
+  nginx:
+    build: ./nginx
+    image: nginx
+    container_name: nginx
+    ports:
+      - "80:80"
+    restart: always
+    depends_on: 
+      - mainapp
+    healthcheck:
+      test: ["CMD","curl","-f","http://mainapp:8080"]
```

yamllint

[warning] too few spaces after comma

(commas)